controllers/routing/routing_controller.go

```diff
@@ -51,7 +51,7 @@ type PlatformRoutingController struct {
 // +kubebuilder:rbac:groups="route.openshift.io",resources=routes,verbs=*
 // +kubebuilder:rbac:groups="networking.istio.io",resources=virtualservices,verbs=*
 // +kubebuilder:rbac:groups="networking.istio.io",resources=gateways,verbs=*
-// +kubebuilder:rbac:groups="networking.istio.io",resources=destinationrule,verbs=*
+// +kubebuilder:rbac:groups="networking.istio.io",resources=destinationrules,verbs=*
```

Comment:
I think it deserves its own fix PR.

Reply:
Oh yea that just got sucked in it seems.

Reply:
reverted https://github.com/maistra/odh-platform/pull/49/commits/a34fec89faca6b5c0f87bbf6e04de6f2e6ec2b9e

Reply:
Do a PR :)